src/winscard_clnt.c

```diff
@@ -327,6 +327,7 @@ struct _psContextMap
 typedef struct _psContextMap SCONTEXTMAP;
 
 static list_t contextMapList;
+static short contextMapListIsInit = 0;
```

Comment:
Why do you need to replace `isExecuted` by `contextMapListIsInit`?

What is the problem with checking the value of `contextMapList` in the destructor?

Reply:
I try to give a variable more clear name...